content/applications/sales/sales/send_quotations/create_quotations.rst

```diff
@@ -0,0 +1,332 @@
+:guilabel:`Quotations` dashboard.
+
+.. important::
+   The :guilabel:`New` button is **only** present if the :guilabel:`Quotations` dashboard is in list
+   or Kanban view.
+
+Clicking the :guilabel:`New` button reveals a blank quotation form, with various fields and tabs to
+configure.
+
+.. image:: create_quotations/quotation-form.png
+   :align: center
+   :alt: A typical quotation form in the Odoo Sales application.
+
+Begin by entering the customer's name in the :guilabel:`Customer` field at the top of the form. This
+is a **required** field.
+
+If the customer's information is already in the database, the :guilabel:`Invoice Address` and
+:guilabel:`Delivery Address` fields auto-populate with the saved information for those respective
+fields, based on the data from that customer's contact record (found in the *Contacts* application).
+
+If the customer was referred by another customer or contact, enter their name in the
+:guilabel:`Referrer` field.
+
+If a :guilabel:`Referrer` is selected, a new field, :guilabel:`Commission Plan` appears, in which
+a commission can be selected from the drop-down menu. This commission is rewarded to the contact
+selected in the :guilabel:`Referrer` field.
+
+Next, if they have not already been auto-populated with the customer's information, enter the
+appropriate addresses in the :guilabel:`Invoice Address` and :guilabel:`Delivery Address` fields.
+Both of these fields are **required**.
+
+Then, if desired, choose a :guilabel:`Quotation Template` from the drop-down field to apply to this
+quotation.
+
+The default date that appears in the :guilabel:`Expiration` field is based on the number configured
+in the :guilabel:`Default Quotation Validity` setting (in :menuselection:`Sales app -->
+Configuration --> Settings`).
+
+.. tip::
+   When using a quotation template, the date in the :guilabel:`Expiration` field is based off the
+   :guilabel:`Quotation Validity` figure on the template form.
+
+If the quotation is for a recurring product or subscription, select the desired :guilabel:`Recurring
+Plan` from that specific drop-down menu.
+
+If desired, select a specific :guilabel:`Pricelist` to be applied to this quotation.
+
+Lastly, select any specific :guilabel:`Payment Terms` to be used for this quotation.
+
+Order Lines tab
+---------------
+
+The first tab on the quotation form is the :guilabel:`Order Lines` tab.
+
+In this tab, select products, and quantities of those products, to add them to the quotation.
+
+There are two ways to add products to the quotation from this tab.
+
+Clicking :guilabel:`Add a product`, selecting the desired item from the :guilabel:`Product`
+drop-down field, and proceeding to adjust the quantity of that selected product, if necessary, is
+one way to add products to a quotation.
+
+Or, click :guilabel:`Catalog` to reveal a separate page, showcasing every item (and every potential
+product variant) in a neatly organized display, with items organizable by :guilabel:`Product
```

Comment:
```suggestion
Or, click :guilabel:`Catalog` to reveal a separate page, showcasing every item, and every potential
product variant, in a neatly organized display, with items organizable by :guilabel:`Product
```

Reply:
Also, I am not sure if I am correct- but it appears to me that these are organized alphabetically by the Internal Reference code on the product form. I checked in a runbot, and it seems to be the case. You can filter with categories and attributes, but I do not think it is actually organizing them in that order?

Reply:
When I mention that it is "organizable" that was intended to imply that users can *choose* to organize the results on the catalog page using the sidebar options. Maybe you mistakenly read that as "**is** organized by..." 🤷